src/components/floating/floating.svelte

```diff
@@ -60,13 +60,19 @@
     return result
   }
 
+  function hasInvalidArrowPosition({ x, y }: { x?: number; y?: number }) {
+    return (!x && !y) || x <= 1 || y <= 1
```

Comment:
Does this work with just
```suggestion
    return !(x || y)
```
or is it important that  we check the `<=` 1?

Reply:
It doesn't actually, that's what I had initially too, the middleware returns a 1 occasionally too, particularly on the y axis.

But with the check for 1 or less I didn't see anymore results that didn't make sense from the middleware

Reply:
Good enough for me - maybe add a comment explaining why we need this check and let's merge.

Reply:
Done!